scripts/qa_crowbarsetup.sh

```diff
@@ -4158,7 +4167,7 @@ function onadmin_run_cct()
             develcloud5|GM5|GM5+up)
                 checkout_branch=cloud5
                 ;;
-            develcloud6)
+            develcloud6|develcloud7)
```

Comment:
we only want to run it in the develcloud case not in the susecloud case (I forgot why)